### PR TITLE
nit: documentation fix

### PR DESCRIPTION
**Pull request by rpriyanshu9**
Signed-off-by: Priyanshu Raj <prraj@redhat.com>

# py-mtcli

## Description

Short description of the change:

documentation change related to #196

### CODE REVIEW

File: docs/tenants/zz_metadata_schema_generated.md [hunk 1]
```diff
@@ -135,7 +135,7 @@
 
 - **`indexImage`** *(string)*
 
-- **`addonImageSetVersion`** *(string)*: A string which specifies the imageset to use. Can either be 'latest' or a version string).
+- **`addonImageSetVersion`**: A string which specifies the imageset to use. Can either be 'latest' or a version string.
```
Comment (garrettthomaskth) [inline docs/tenants/zz_metadata_schema_generated.md:138]:
```suggestion
- **`addonImageSetVersion`** *(string)*: Specifies the imageset to use. Can either be 'latest' or a version string.
```

Comment (garrettthomaskth) [reply docs/tenants/zz_metadata_schema_generated.md:138]: The might be a little more consistent with the structure used for the other fields

Comment (rpriyanshu9) [reply docs/tenants/zz_metadata_schema_generated.md:138]: thanks for the review. :) I get your point, but this corresponds to the change that Amador did in #196, where `type: string` was removed from the `metadata.schema.yaml` (https://github.com/mt-sre/managed-tenants-cli/pull/196/commits/73ef3689450f453f86e84e5ed0b6cf4692c4cd73#diff-ad3eac1137a13a51306b5b5493dc7723592bf3415b304eae3b5eaece2733fb81L267). 

This is an auto-generated change, if you run `make generate`, you'll also get this change. I got this in a different PR, and this change did not align with the PR's context. So I decided to create another PR for this.

Comment (garrettthomaskth) [reply docs/tenants/zz_metadata_schema_generated.md:138]: Could you maybe also change it in `managedtenants/schemas/metadata.schema.yaml`? :)

Comment (rpriyanshu9) [reply docs/tenants/zz_metadata_schema_generated.md]: done!

**Review by garrettthomaskth — APPROVED** [review]
lgtm

**Pull request merged**